engine/platform/src/platform_window_glfw3.cpp

```diff
@@ -481,7 +480,7 @@ namespace dmPlatform
 
     int32_t GetMouseWheel(HWindow window)
     {
-        return (int32_t) window->m_MouseScrollY;
+        return (int32_t) round(window->m_MouseScrollY);
```

Comment:
This is what the old GLFW implementation did with the double values. Seems to have most impact on mac, on windows its works the same regardless of the rounding